src/file.js

```diff
@@ -0,0 +1,27 @@
+const assert = require('assert')
+
+/**
+ * Names of files in a Bitbucket repo that we may want to know about. We list
+ * them explicitly so that we can make fewer API calls to Bitbucket when looking
+ * for files (i.e. fewer pages of results to separately request from Bitbucket).
+ *
+ * @type {string[]}
+ */
+const fileNamesOfInterest = [
```

Comment:
This `fileNamesOfInterest` was (basically) only needed for optimizing API calls to Bitbucket. How have you found it useful when dealing with a local repo? It looks like you have, I just haven't fully followed all the details of how.

Reply:
Probably not strictly necessary, mainly used to limit number of filenames stored in an array.

Reply:
Okay. Would you mind updating the comment above, then, to broaden the purpose statement?